src/main/java/net/sf/jabref/JabRefGUI.java

```diff
@@ -192,6 +196,19 @@ private void openWindow() {
         if (!bibDatabases.isEmpty()) {
             JabRefGUI.getMainFrame().getCurrentBasePanel().getMainTable().requestFocus();
         }
+
+        boolean isSharedDatabaseEdited = Globals.prefs.getBoolean(JabRefPreferences.SHARED_DATABASE_LAST_EDITED);
+        if (isSharedDatabaseEdited) {
+            boolean isFocused = Globals.prefs.getBoolean(JabRefPreferences.SHARED_DATABASE_LAST_FOCUSED);
+            String keywordSeparator = Globals.prefs.get(JabRefPreferences.KEYWORD_SEPARATOR);
+
+            try {
+                new SharedDatabaseUIManager(mainFrame, keywordSeparator).openLastSharedDatabaseTab(isFocused);
+            } catch (ClassNotFoundException | SQLException | DatabaseNotSupportedException e) {
+                LOGGER.error("Failed to restore shared database. Use connection dialog to connect.");
```

Comment:
I intentionally didn't add it, cause those exceptions basically inform the user about the failure. It's not a real problem. But I think it's better to use `LOGGER.info(...)`.